pkg/license/license.go

```diff
@@ -0,0 +1,53 @@
+// Package license reads a license file and
```

Comment:
I don't think we need this package. Can you delete it please?

ioutil.Readfile is fine.

Reply:
Noted! 
@alexellis, I will make the changes as recommended. Thanks.